This patch is to remove the Warning "componentWillReceiveProps has been renamed and not recommended for use" and "useNativeDriver was not specified"

diff --git a/node_modules/react-native-progress-bar-animated/src/AnimatedProgressBar.js b/node_modules/react-native-progress-bar-animated/src/AnimatedProgressBar.js
index 29a0f82..9a8922a 100644
--- a/node_modules/react-native-progress-bar-animated/src/AnimatedProgressBar.js
+++ b/node_modules/react-native-progress-bar-animated/src/AnimatedProgressBar.js
@@ -26,8 +26,8 @@ class ProgressBar extends React.Component {
     }
   }
 
-  componentWillReceiveProps(props) {
-    if (props.value !== this.state.progress) {
+  static getDerivedStateFromProps (props,state) {
+    if (props.value !== state.progress) {
       if (props.value >= 0 && props.value <= this.props.maxValue) {
         this.setState({ progress: props.value }, () => {
           if (this.state.progress === this.props.maxValue) {
@@ -40,6 +40,7 @@ class ProgressBar extends React.Component {
         });
       }
     }
+    return null;
   }
 
   componentDidUpdate(prevProps) {
@@ -61,6 +62,7 @@ class ProgressBar extends React.Component {
       easing: Easing[this.props.barEasing],
       toValue: toValue > 0 ? toValue : 0,
       duration: this.props.barAnimationDuration,
+      useNativeDriver: false 
     }).start();
   }
 
